app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -234,6 +223,25 @@ def is_ineligible_or_unsupported_for_property_tax
         StateFile::NjHomeownerEligibilityHelper.determine_eligibility(@intake) != StateFile::NjHomeownerEligibilityHelper::ADVANCE
       end
 
+      def calculate_line_40a
+        case @intake.household_rent_own
+        when "own"
+          if @intake.property_tax_paid.nil?
+            return nil
+          end
+          property_tax_paid = @intake.property_tax_paid
+        when "rent"
+          if @intake.rent_paid.nil?
+            return nil
+          end
+          property_tax_paid = @intake.rent_paid * RENT_CONVERSION
+        else
+          return nil
+        end
+
+        is_mfs_same_home ? (property_tax_paid / 2.0).round : property_tax_paid.round
```

Comment:
You may consider `2.to_d` since this is money. I'm not sure that it would matter with a single calculation that's being rounded anyways, but converting to decimal is more correct because the decimal type uses (to my understanding) two integers to represent the decimal in a fraction notation until the number is ultimately intended to be used when it finally does the math that converts back to a float for display.

In this specific case it probably doesn't matter but it might be good to begin using decimals by default with taxes?

Reply:
This was not part of this PR, I just moved the method to put them in sequential order